source/shared/smart-contracts/guides/custom-errors.rst

```diff
@@ -55,3 +55,25 @@ Return custom errors, as you would with any other error type:
        _ctx: &impl HasReceiveContext<()>,
        _state: &mut State,
    ) -> Result<A, MyError> { Err(MyError::ErrTwo) }
+
+Standard error codes
+====================
+
+The following standard error codes exist:
+
+=========================================== ===========================
+Variant                                     Error code
+=========================================== ===========================
+()	                                         i32::MIN + 1 (-2147483647)
+ParseError	                                i32::MIN + 2 (-2147483646)
+LogError::Full	                             i32::MIN + 3 (-2147483645)
+LogError::Malformed	                       i32::MIN + 4 (-2147483644)
+NewContractNameError:://! MissingInitPrefix i32::MIN + 5 (-2147483643)
```

Comment:
```suggestion
NewContractNameError::MissingInitPrefix                 i32::MIN + 5 (-2147483643)
```

Reply:
You are welcome to update the documentation to include the links to the errors. I was simply inserting what you added to the original issue.